core/service/power-management-service.js

```diff
@@ -0,0 +1,39 @@
+var Montage = require("montage").Montage,
+    FreeNASService = require("core/service/freenas-service").FreeNASService,
+    Model = require("core/model/model").Model;
+
+var PowerManagementService = exports.PowerManagementService = Montage.specialize({
+    _instance: {
+        value: null
+    },
+
+    _dataService: {
+        value: null
+    },
+
+    listDrivers: {
+        value: function () {
+            return Model.populateObjectPrototypeForType(Model.ServiceUps).then(function (ServiceUps) {
```

Comment:
You should move this in a separate property to avoid calling it each time.
Something like
```
_serviceUpsServices: {
    get: function() {
        return this._serviceUpsServicesPromise || this._serviceUpsServicesPromise = Model.populateObjectPrototypeForType(Model.ServiceUps);
    }
}
````